test/test-utils/matchers/to-have-hp.ts

```diff
@@ -1,22 +1,48 @@
 import { getPokemonNameWithAffix } from "#app/messages";
 import type { Pokemon } from "#field/pokemon";
 import { isPokemonInstance, receivedStr } from "#test/test-utils/test-utils";
+import { toDmgValue } from "#utils/common";
 import type { MatcherState, SyncExpectationResult } from "@vitest/expect";
 
+/** Options type for {@linkcode toHaveHp} */
+export interface toHaveHpOptions {
```

Comment:
```suggestion
export interface ToHaveHpOptions {
```
Types/interfaces should be PascalCase.